partridge.js

```diff
@@ -61,6 +61,18 @@ function psquares_init(id, config) {
             psquares_place_piece(data[i][1]+(data[i][0]-1)/2, data[i][2]+(data[i][0]-1)/2);
         }
     }
+    document.addEventListener(
+        "keydown",
+        (event) => {
+          const keyName = event.key;
```

Comment:
const doesn't work in old browsers (and I think in one of the online ones
```suggestion
          var keyName = event.key;
```